content/behind-atom/sections/keymaps-in-depth.asciidoc

```diff
@@ -95,6 +95,21 @@ image::../../images/keybinding.png[keybinding resolver]
 
 If you want to force the native browser behavior for a given keystroke, use the `native!` directive as the command of a binding. This can be useful to enable the correct behavior in native input elements, for example. If you apply the `.native-key-bindings` class to an element, all the keystrokes typically handled by the browser will be assigned the `native!` directive.
 
+```html
+<form>
+  <input type="text" />
+  <input type="text" />
+</form>
+```
+
+```coffeescript
+  "atom-workspace":
+    "ctrl-alt-o": "your-package's-name:toggle"
+  ".your-package's-root-element-class":
+    "enter": "native!"
+    "backspace": "native!"
```

Comment:
I tried that before resorting to `native!`. It seemed to break tab (tabbing between fields).